### PR TITLE
Technomancer Exultant Hardsuit: Art improvements

### PR DESCRIPTION
**Pull request by NanakoAC**
This PR builds on a recent one that seperated the exultant hardsuit's gloves and boots off into seperate objects.

Now those seperate objects have new sprites. Including advanced white magboots ported from TG
A picture is worth a thousand words
What's an animated gif worth?

And also heres how it looks if you toggle off the chestpiece
topless and lewd

### CODE REVIEW

**Comment by IlyaIII** [comment]
Is it just me or gif is slightly broken?

**Comment by NanakoAC** [comment]
It was broken, i fixed it.
Imgur likes to upload them as .gifv for some reason which won't embed, but removing the v from the end makes it work fine

File: code/modules/admin/bluespacetech.dm [hunk 1]
```diff
@@ -193,6 +193,15 @@ ADMIN_VERB_ADD(/client/proc/cmd_dev_bst, R_ADMIN|R_DEBUG, TRUE)
 	status_flags ^= GODMODE
 	src << span("notice", "God mode is now [status_flags & GODMODE ? "enabled" : "disabled"]")
 
+/mob/living/carbon/human/bst/verb/allow_unequip()
```
Comment (NanakoAC) [inline code/modules/admin/bluespacetech.dm]: I've been meaning to add this verb for a while, this PR gave me a reason to do it.

Allows BSTs to take off their items, useful when debugging worn equipment

File: code/modules/clothing/shoes/magboots.dm [hunk 2]
```diff
@@ -6,15 +6,18 @@
 	force = 3
 	overshoes = 1
 	var/magpulse = 0
+	var/mag_slow = 3
 	var/icon_base = "magboots"
 	action_button_name = "Toggle Magboots"
 	var/obj/item/clothing/shoes/shoes = null	//Undershoes
 	var/mob/living/carbon/human/wearer = null	//For shoe procs
+	armor = list(melee = 40, bullet = 30, laser = 30,energy = 25, bomb = 50, bio = 100, rad = 70)
```
Comment (NanakoAC) [inline code/modules/clothing/shoes/magboots.dm]: Magboots are big, heavy and slow as footwear goes, it seems reasonable to give them some protection. The slowdown is the obvious drawback

File: code/modules/clothing/spacesuits/rig/suits/station.dm [hunk 3]
```diff
@@ -115,7 +138,23 @@
 /obj/item/clothing/gloves/rig/ce
 	name = "insulated gloves"
 	siemens_coefficient = 0
+	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE
+	armor = list(melee = 50, bullet = 40, laser = 40,energy = 25, bomb = 60, bio = 100, rad = 100)
```
Comment (spaceplaceexplorer) [inline code/modules/clothing/spacesuits/rig/suits/station.dm]: All armor statistics like max_heat_protection_temperature and armor values are inherited from the weapon/rig item. They have no effect when defined in a rig piece except for siemens_coefficient. See line 141 in rig.dm.

File: code/modules/clothing/spacesuits/rig/suits/station.dm [hunk 4]
```diff
@@ -115,7 +138,23 @@
...
+
+/obj/item/clothing/shoes/magboots/rig/ce
+	name = "Advanced Magboots"
+	desc = "Advanced magnetic boots that have a lighter magnetic pull, placing less burden on the wearer."
+	mag_slow = 2
+	icon_state = "advmag0"
+	icon_base = "advmag"
+	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE
+	armor = list(melee = 50, bullet = 40, laser = 40,energy = 25, bomb = 60, bio = 100, rad = 100)
```
Comment (spaceplaceexplorer) [inline code/modules/clothing/spacesuits/rig/suits/station.dm]: Same here.

**Pull request merged**